content-src/components/Base/Base.js

```diff
@@ -15,6 +15,8 @@ const Base = React.createClass({
 
     this.props.dispatch(actions.RequestSearchState());
 
+    this.props.dispatch(actions.RequestHighlightsLinks());
```

Comment:
You should remove line 12,  `this.props.dispatch(actions.RequestFrecentLinks())`, since it's not being used anymore